src/Notifier.php

```diff
@@ -51,6 +58,9 @@ public function __construct($opt = [])
                 return $this->rootDirectoryFilter($notice);
             });
         }
+
+        $handler = (isset($this->opt['httpClient']) ? $this->opt['httpClient'] : null);
```

Comment:
this would trigger a notice:
`
php > $test = [];
php > echo $test['does-not-exist'];
PHP Notice:  Undefined index: does-not-exist in php shell code on line 1
`
In PHP7 we could use the null coalescing operator instead of isset()